### PR TITLE
add a lot more to "venue" and call it "attending elm-conf"

### PR DESCRIPTION
**Pull request by BrianHicks**
@lukewestby would you mind looking at this when you get back? No rush. I'm just a bit concerned about the FAQ part, and wonder if you have any more things you've been asked.

### CODE REVIEW

File: content/attend.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+title: Attending elm-conf
+---
+
+elm-conf will be taking place on September 26, 2018 at the historic Peabody Opera House in St. Louis, MO.
```
Comment (lukewestby) [inline content/attend.md]:
```
- elm-conf will be taking place on September 26
+ elm-conf will take place on September 26
```

File: content/attend.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+|:-|-:|:-|
+| Metrolink Train | $3 | Disembark at the Union Station Stop |
+| Lyft | $28-37 | Get updated estimates at [Lyft's Fare Estimator](https://www.lyft.com/fare-estimate) |
+| Taxi | ~$40 | |
+
+## Staying in St. Louis
+
+Hotel stays can be booked in the Strange Loop blocks at Union Station (~10 minute walk) or the Ballpark Hilton (~15 minute walk.)
+See [their hotel page](https://thestrangeloop.com/register.html) for more details.
+Strange Loop will be providing shuttle service from Union Station and Hilton Ballpark to the conference and party.
```
Comment (lukewestby) [inline content/attend.md]:
```
- Strange Loop will be providing
+ Strange Loop will provide
```

File: content/attend.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+It's a small group of people who work together to advance Elm, and who are taking personal responsibility for the future of the language.
+
+### Who are these people referred to by first name in talks?
+
+- **"Evan"** is probably Evan Czaplicki, creator of Elm.
+- **"Richard"** is probably Richard Feldman, author of *Elm in Action*. [Here's his talk](speakers/richard-feldman.md).
+- **"Noah"** is probably `@eeue56` on the Elm Slack, a prolific Elm contributor.
+- **"Tereza"** is probably Tereza Sokol, creator of `elm-plot`. [Here's her talk](speakers/tereza-sokol.md).
+- **"Aaron"** is probably Aaron VonderHaar, creator of `elm-format`.
+- **"Luke"** is probably Luke Westby, creator of [Ellie](https://www.ellie-app.com). (TODO: verify www)
```
Comment (lukewestby) [inline content/attend.md]: 👍 no www

File: content/attend.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+It's worth your time!
+
+### Why are people avoiding the word "components"?
+
+Since "component" means "React component" to many people, it's generally easier to communicate if we avoid using this term when we mean "a smaller part of a larger application."
+Try using "widget" or "doodad" instead.
+
+### What's "the core team"?
+
+It's a small group of people who work together to advance Elm, and who are taking personal responsibility for the future of the language.
```
Comment (lukewestby) [inline content/attend.md]: Shoutouts to Personal Responsibility™️

File: content/attend.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+But it's not always easy to introduce yourself and strike up a conversation!
+Because of this, we will be doing a number of things to get people talking.
+Keep an eye out for these things on the conference day!
+
+### Getting Around
+
+For getting *to* the conference, see the information higher up on this page.
+As for getting *around* the venue, we plan to do a quick venue tour on the morning of the conference to locate the restrooms, quiet room, and other points of interest.
+
+## Frequently Asked Questions
```
Comment (lukewestby) [inline content/attend.md]: This is v good

File: content/attend.md (hunk 4 above)
Comment (BrianHicks) [reply content/attend.md]: 😅 too strong you think? I don’t feel I have a great way to describe it.

Comment (lukewestby) [reply content/attend.md]: no it's good how it is, just making a shout out

Comment (BrianHicks) [reply content/attend.md]: shout out to shouts out.

**Pull request merged**